app/views/layouts/application.html.erb

```diff
@@ -7,6 +7,16 @@
     <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
     <title>Official Membership Register | Ruby New Zealand Incorporated</title>
 
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
+
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha384-HSMxcRTRxnN+Bdg0JdbxYKrThecOKuH5zCYotlSAcp1+c8xmyTe9GYg1l9a69psu" crossorigin="anonymous">
+
+    <!-- Optional theme -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap-theme.min.css" integrity="sha384-6pzBo3FDv/PJ8r2KRkGHifhEocL+1X2rVCTTkUfGk7/0pbek5mMa1upzvWbrUbOZ" crossorigin="anonymous">
+
+    <!-- Latest compiled and minified JavaScript -->
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha384-aJ21OjlMXNL5UyIl/XNwTMqvzeRMZH2w8c5cRVpzpU8Y5bApTppSuUkhZXN0VxHd" crossorigin="anonymous"></script>
```

Comment:
What about just adding these to webpacker, so you can use yarn to manage these dependencies?

Reply:
> What about just adding these to webpacker, so you can use yarn to manage these dependencies?

To tell you the truth the app is so small that I believe it is better to use a CDN instead of making it use time and resources to keep the assets "in house".

Reply:
Your call here, but I think there are a few downsides to the CDN approach:

1. We have to look in multiple places to find which dependencies we're relying on
2. If one of our dependencies contains an XSS vulnerability, we won't get any GitHub security notifications about it
3. Upgrading is a manual process, as opposed to being able to `yarn upgrade`

Reply:
> Your call here, but I think there are a few downsides to the CDN approach:
> 
> 1. We have to look in multiple places to find which dependencies we're relying on
> 2. If one of our dependencies contains an XSS vulnerability, we won't get any GitHub security notifications about it
> 3. Upgrading is a manual process, as opposed to being able to `yarn upgrade`

Yes, I totally agree, let's make a deal: let's merge this PR using a CDN and then I will work on that as the next feature.

This way we can deploy something fast and fix production :wink:.

What do you think?

Reply:
Sounds good 😊